index.ts

```diff
@@ -23,6 +24,11 @@ createConnection({
   database: Config.database.name,
   entities,
   logging: Config.isDevelopment,
+  cache: {
+    provider(_connection) {
+      return new InMemoryDatabaseCache();
+    },
+  },
```

Comment:
should this be `InMemoryDatabaseCache.getInstance()`? doesn't seem like `getInstance()` and the singleton instance are being used anywhere else

Reply:
TypeORM probably uses those somewhere since `InMemoryDatabaseCache` extends the TypeORM cache interface.